chall03/vsempere.py

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env python3
+import sys
+
+def printUsage():
+    print('usage: ./vsempere.py <1-9 squared_rows...>');
+
+def getOffset(direction, board_size):
+    if direction == 0:
+        return 1;
+    if direction == 1:
+        return board_size;
+    if direction == 2:
+        return -1;
+    return -board_size;
+
+def buildArray(argv):
+    if (len(argv)<=1 or len(argv)>=10):
```

Comment:
So you didn't understand correctly. We specified `NOTE: A valid square can range from a size of 1*1 to a size of 20*20.`.
The part in the usage that says `<1-9 squared_rows...>` means that the only valid characters are ascii 1 to 9. 

(Even if you understood that the biggest valid test was 9*9, your script still prints usage in a 9*9 instead of solving it).